controllers/suite_test.go

```diff
@@ -55,18 +53,18 @@ var _ = BeforeSuite(func(done Done) {
 
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "config", "crd", "bases")},
+		CRDDirectoryPaths: []string{
+			filepath.Join("..", "config", "crd", "bases"),
+			filepath.Join("..", "capi-config", "crds"),
```

Comment:
You could use the vendored directory, but I see no vendor in this repo, this might need to get documented